### PR TITLE
DOC simplify the readme

### PR DESCRIPTION
**Pull request by rth**
This PR aims to simplify the readme, and improve linking to the documentation for the v1.4 release (#67). In particular, redundant examples are removed instead pointing to the example gallery.

The updated version can be viewed in https://github.com/rth/PyKrige/blob/simplify-readme/README.rst

Please let me know what you think.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -21,28 +21,36 @@ Kriging Toolkit for Python
...
 
-To update PyKrige from PyPi, type the following into a command line.
+scikit-learn is an optional dependency needed for parameter tuning and regression kriging.
+
+
+If you use conda, PyKrige can be installed from the `conda-forge` channel with,
 
 .. code:: bash
 
-    pip install --upgrade pykrige
```
Comment (rth) [inline README.rst]: pip prior to mid 2017 by default has a eager upgreade strategy  https://github.com/pypa/pip/commit/375ed4d3fcf2679dfcd011c3441caa05e5b5c94a  that can break things in the existing packages https://github.com/pypa/pip/issues/304 so I'd rather not put this in the readme.

**Comment by bsmurphy** [comment]
I'd prefer to keep the UK and 3D examples since they demonstrate slightly different things than just regular OK. Can we stick those over in the examples? You (or I) can just copy-paste the UK and 3D example codes into new files, maybe add a plot (if you feel like it).

**Comment by rth** [comment]
Sorry for the delay @bsmurphy . I added back the examples (that can also be found in http://pykrige.readthedocs.io/en/latest/examples/index.html).

**Comment by rth** [comment]
Also updated the long description to contain the full readme and updated appveyor. 

Merging, @bsmurphy please edit the readme on master if there are changes you are not enthousiastic about. Links to the stable docs are not working yet, it's necessary to create a git tag for readthedocs to generate a link to it.

**Pull request merged**